src/GaussianProcess.jl

```diff
@@ -51,13 +53,13 @@ $(DocStringExtensions.FIELDS)
 """
 struct GaussianProcess{GPPackage} <: MachineLearningTool
     "the Gaussian Process (GP) Regression model(s) that are fitted to the given input-data pairs"
-    models::Vector
+    models::Vector{Union{<:GPE, <:PyObject, Nothing}}
```

Comment:
I think we need to make it more clear in this file where `GPE` comes from - I think it is within `GaussianProcesses.jl`? - so perhaps could we add some description/comment  here about it

Reply:
Replaced all occurrences of GPE with the fully qualified form GaussianProcesses.GPE in 
b47ba89, hopefully this should make the code self-explanatory.